pkg/blobserver/diskpacked/diskpacked.go

```diff
@@ -125,9 +125,26 @@ func IsDir(dir string) (bool, error) {
 // files.
 func New(dir string) (blobserver.Storage, error) {
 	var maxSize int64
-	if ok, _ := IsDir(dir); ok {
-		// TODO: detect existing max size from size of files, if obvious,
-		// and set maxSize to that?
+	var n, atMax int
```

Comment:
`n` could have a better name. What is it counting? How about numPacks?

Also I think that  the `var n, atMax int` can be inside the `if` statement. It is only used in that case.